control-plane/agents/src/bin/core/tests/volume/snapshot.rs

```diff
@@ -417,3 +417,151 @@ async fn snapshot_timeout() {
     let tx = snapshot.meta().transactions().get("2").unwrap();
     assert_eq!(tx[0].status().to_string().as_str(), "Created");
 }
+
+#[tokio::test]
+async fn unknown_snapshot_garbage_collector() {
+    let cluster = ClusterBuilder::builder()
+        .with_rest(false)
+        .with_agents(vec!["core"])
+        .with_io_engines(1)
+        .with_pools(1)
+        .with_cache_period("500ms")
```

Comment:
this would be "slow", make it very small ms, especially since we're running at 5x here.

Reply:
Indeed. maybe 50ms?